SliderButton/SliderButton.js

```diff
@@ -57,14 +58,15 @@ const SliderProgress = kind({
 	name: 'SliderProgress',
 	propTypes: {
 		css: PropTypes.object,
-		values: PropTypes.arrayOf(PropTypes.number)
+		values: PropTypes.arrayOf(PropTypes.string)
 	},
 	styles: {
 		css: componentCss,
 		className: 'track'
 	},
 	render: ({children, css, values, ...rest}) => {
-
+		delete rest.progressAnchor;
+		delete rest.backgroundProgress;
```

Comment:
Looks like we need to clean up `ui/Slider` to avoid unnecessary deleting props both here and the slider knob.

Reply:
I think the API "contract" established in `ui/Slider` is appropriate. Tooltips and `backgroundProgress` etc do make sense to forward into these specialized components (in any theme). This component in particular, `SliderButton` is leveraging several of the qualities of `ui/Slider` but it is also providing its own components for the building blocks (internal components `SlideKnob` and `SliderProgress`). Each of those building blocks needs to respect its role and handle the incoming props accordingly, so it doesn't warn or break. This seems fine to me.

I've added linting ignore lines for these props so we don't have to name them in the proptypes. _It always seems weird to declare a prop, just to delete it later without doing anything with or to it._

Reply:
We might consider declaring these as prop types but marking them `@private` with docs that indicate why they exist. That'll give us better traceability for the rationale (which i think is sound) than an otherwise undocumented `eslint-disable` comment.

Reply:
If at very least, the docs for these could read:
> Unused, deleted in render.

I guess that's good practice; it just seems cumbersome.

Reply:
I'm most concerned with clarity for the reader. I think that describes _what_ and not _why_. The _what_ is relatively clear from the code but the _why_ isn't. I think I'd favor some text that clarifies that these props are required by the `ui/Slider` API but are not required for this implementation.

Reply:
I follow now. That makes sense.